css/style.css

```diff
@@ -0,0 +1,44 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+#main{
+    background-color: pink;
+    width: 80%;
```

Comment:
An user experience is pretty important once we deliver a page.
If we change the width of the line 9 to 100%, it improves visually the site.
Please kindly change the line 9.